### PR TITLE
Improving community guidelines

### PR DESCRIPTION
**Pull request by glatard**
This PR improves our community guidelines based on [GitHub's suggestions](https://github.com/CONP-PCNO/conp-dataset/community). More specifically, it adds:
* A code of conduct, identical to [the one in conp-portal](https://github.com/CONP-PCNO/conp-portal/blob/master/.github/CODE_OF_CONDUCT.md).
* A contribution guide, describing how to add a dataset, including the forking and PR procedure. The corresponding instructions in README.md have been updated accordingly. README.md now focuses on instructions on how to use the dataset.
* Issue templates, identical to the ones in conp-portal.

A PR template was already proposed in #50. Licensing will be discussed in a separate issue.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# Contribution			
+
+New contributions are more than welcome! ❤️ 
+
+## Contribution workflow
+
+Whether you are planning to add
+a new dataset, add data to an existing one, or simply fix a typo or a bug
+wherever in this repository, you will have to follow the following workflow:
+
```
Comment (shots47s) [inline .github/CONTRIBUTING.md]: Would re-write:

Contributors should perform the workflow below if:
1. you are planning to add a new dataset
2. you are adding data to an already existing dataset
3. or make any changes (e.g. fixing a typo or a bug)

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+1. Fork this repository, or make that your existing fork is up-to-date.
```
Comment (shots47s) [inline .github/CONTRIBUTING.md]: Add...

"Please perform the following steps:"

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+5. Open a pull request (PR) to the main repository.
+For steps 3. and 4., see detailed instructions in the sections below.
+
+This follows a very common GitHub workflow. If you are not familiar with
+it, we suggest to read [GitHub documentation](https://help.github.com/en),
+in particular the articles on [forks]
+(https://help.github.com/en/articles/working-with-forks) and [pull
+requests](https://help.github.com/en/articles/creating-a-pull-request-from-a-fork).
+
+Your PR will be reviewed by two members of our technical team. They will
```
Comment (shots47s) [inline .github/CONTRIBUTING.md]: "...by at least two members..."

**Comment by jbpoline** [comment]
pending the language improvements, sounds great, this was needed - thanks Tristan

**Pull request merged**